File: src/VisualStudio/CSharp/Impl/Options/AutomationObject/AutomationObject.SymbolSearch.cs
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.

using Microsoft.CodeAnalysis.SymbolSearch;

namespace Microsoft.VisualStudio.LanguageServices.CSharp.Options
{
    public partial class AutomationObject
    {
        public int AddImport_SuggestForTypesInReferenceAssemblies
        {
            get { return GetBooleanOption(SymbolSearchOptionsStorage.SearchReferenceAssemblies); }
            set { SetBooleanOption(SymbolSearchOptionsStorage.SearchReferenceAssemblies, value); }
        }

        public int AddImport_SuggestForTypesInNuGetPackages
        {
            get { return GetBooleanOption(SymbolSearchOptionsStorage.SearchNuGetPackages); }
            set { SetBooleanOption(SymbolSearchOptionsStorage.SearchNuGetPackages, value); }
        }
    }
}

File: src/VisualStudio/VisualBasic/Impl/Options/AutomationObject/AutomationObject.SymbolSearch.vb
' Licensed to the .NET Foundation under one or more agreements.
' The .NET Foundation licenses this file to you under the MIT license.
' See the LICENSE file in the project root for more information.

Imports Microsoft.CodeAnalysis.SymbolSearch

Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Options
    Partial Public Class AutomationObject
        Public Property Option_SuggestImportsForTypesInReferenceAssemblies As Boolean
            Get
                Return GetBooleanOption(SymbolSearchOptionsStorage.SearchReferenceAssemblies)
            End Get
            Set(value As Boolean)
                SetBooleanOption(SymbolSearchOptionsStorage.SearchReferenceAssemblies, value)
            End Set
        End Property

        Public Property Option_SuggestImportsForTypesInNuGetPackages As Boolean
            Get
                Return GetBooleanOption(SymbolSearchOptionsStorage.SearchNuGetPackages)
            End Get
            Set(value As Boolean)
                SetBooleanOption(SymbolSearchOptionsStorage.SearchNuGetPackages, value)
            End Set
        End Property
    End Class
End Namespace
